src/hapi/pipelines/utilities/population.py

```diff
@@ -0,0 +1,90 @@
+"""Functions specific to the population theme."""
+
+import re
+from logging import getLogger
+from typing import Dict
+
+from sqlalchemy.orm import Session
+
+from hapi.pipelines.database.db_population import DBPopulation
+from hapi.pipelines.utilities.admins import Admins
+from hapi.pipelines.utilities.age_range import AgeRange
+from hapi.pipelines.utilities.gender import Gender
+from hapi.pipelines.utilities.metadata import Metadata
+
+logger = getLogger(__name__)
+
+
+def populate_population(
```

Comment:
This could possibly be made into a class along with the helper functions, although the need to do so is less since there isn't yet any state that is to be used later.

Reply:
Yes I was on the fence about this one. On the one hand I don't really like making things into classes that don't need to be. But on the other hand, making it a class would make it consistent with the other table population utils. @b-j-mills want to be the tie-breaker?

Reply:
A class it shall be!